### DISCUSSION TITLE
Release notes for monorepo

### DISCUSSION THREAD

**Discussion by anees-tricon** [community/community#55283] [Repositories]
### Select Topic Area

Product Feedback

### Body

We work with a monorepo, so a release might be specific to `app1` in the mono repo, so ideally we need a way to create `app1.release.yml` which will create the correct release notes by filtering out the PRs with the label `app1`.

Something like the correct release template will be picked using the Release tag/ title. Basically a monorepo needs more support for automated release notes generation.

I know we can use the query params to create the release and provide the release template path, but we do releases within Github itself ideally so that is not an option.

**Comment by vvakame** [comment]
👍 

I'm considering about monorepo, there is currently no way to generate a good release note there.
For example, we would like a way to successfully generate release notes for individual packages with the following repository structure and tag rules.
https://github.com/googleapis/google-cloud-go/releases

**Comment by The-Funk** [comment]
@vvakame @anees-tricon FYI I found a way to do release notes broken down by project by adding labels but you have to write a lot of excludes if you want any sort of granularity into what is a bugfix vs an enhancement, etc.

I have another ticket open here in support of making that system a little more flexible.

https://github.com/orgs/community/discussions/140445
